src/components/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { CardBack } from './CardBack';
+import { CardFront } from './CardFront';
+
+interface CardProps {
+  Product: Array<any>;
+}
+
+type CardState = {
+  IsSideFront: boolean;
+}
+
+export class Card extends Component<CardProps, CardState> {
```

Comment:
Przepisać do komponentu funkcyjnego z użyciem hooków useState(), pokazywałem wam to.